compiler/src/iree/compiler/Dialect/HAL/Target/LLVMCPU/LLVMCPUTarget.cpp

```diff
@@ -51,6 +51,12 @@ static llvm::cl::opt<bool> clEnableCPUMicrokernels(
         "Enables microkernel lowering for llvmcpu backend (experimental)"),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clLinkCPUMicrokernelsBitcode(
+    "iree-llvmcpu-link-microkernels-bitcode",
+    llvm::cl::desc(
+        "Enables linking microkernel bitcode in the llvmcpu backend"),
```

Comment:
```suggestion
        "Links ukernel bitcode libraries into generated executables."),
```

Reply:
Wait, i was starting to do a `s/microkernel/ukernel/g` in this file, but noticed the existing command-line flag `iree-llvmcpu-enable-microkernels`.  Since it's user-facing and non-default so users are passing it explicitly, the renaming is going to be a bit more of a process.  I'm for now going to only use the word `ukernel` in the new flag, not change the existing one, OK? @benvanik

Reply:
I've pushed with just these review changes, https://github.com/openxla/iree/pull/15354/commits/2613c66c2c1eab71a9cc34d67f5ab89f875c99af
PTAL @benvanik

Reply:
sounds good - we should fix that flag too!